test/powershell/Modules/Microsoft.PowerShell.Management/Resolve-Path.Tests.ps1

```diff
@@ -48,8 +48,11 @@ Describe "Resolve-Path returns proper path" -Tag "CI" {
             Pop-Location
         }
     }
-    It 'Resolve-Path should support user specified base paths' {
+    It 'Resolve-Path should support resolving relative paths with user specified base paths' {
```

Comment:
We should have some test variations:

| Path | RelativeBasePath |
| -- | -- |
| absolute | absolute |
| relative | absolute |
| relative | relative |
| invalid | valid |
| invalid | invalid |